docs/mkdocs/docs/index.md

```diff
@@ -1,387 +1,495 @@
-![](images/FullWithBorder.png)
+ ![](images/FullWithBorder.png)
 
 ## What is ArcticDB?
 
-ArcticDB is an embedded/serverless database engine designed to integrate with Pandas and the Python Data Science ecosystem. ArcticDB enables 
-you to store, retrieve and process DataFrames at scale, backed by commodity object storage (S3-compatible storages and Azure Blob Storage).
+ArcticDB is a serverless DataFrame database engine designed for the Python Data Science ecosystem. 
+
+ArcticDB enables you to store, retrieve and process DataFrames at scale, backed by commodity object storage.
```

Comment:
I would keep the `(S3-compatible storages and Azure Blob Storage)` bit.
Not everyone knows what is an object store.